packages/app/src/app/overmind/namespaces/editor/internalActions.ts

```diff
@@ -426,7 +426,12 @@ export const forkSandbox: AsyncAction<{
       }
     }
 
-    const forkedSandbox = await effects.api.forkSandbox(id, usedBody);
+    const forkedSandbox = await effects.api.forkSandbox(id, {
+      ...(sandbox.title && !sandbox.title.includes('(FORKED)')
+        ? { title: sandbox.title + ' (FORKED)' }
```

Comment:
Love the idea! Can we make it capitalised. All caps in my head is like shouting :P

```suggestion
        ? { title: sandbox.title + ' (Forked)' }
```